### DISCUSSION TITLE
HTTPS using solid-start start?

### DISCUSSION THREAD

**Discussion by hiszd** [solidjs/solid-start#733] [Q&A]
I setup https using vite-plugin-mkcert, but this does not work when I run `npx solid-start build` and `npx solid-start start`
it just runs a http server instead.
Is there a way to use https with the solid-start currently?

**Comment by AlexErrant** [comment]
What adapter are you using? I got [it working with Cloudflare Workers](https://github.com/AlexErrant/Pentive/commit/838cc96086044159d3b91d539c3e25647e0702cc). (See [this](https://github.com/AlexErrant/Pentive/blob/main/design-decisions/setup.md#5-generate-certs) for how I generated the cert.)

**Reply by AlexErrant** [reply]
Quite a bit of the doc above is related to getting HTTPS working with a solid-start project ([`hub`](https://github.com/AlexErrant/Pentive/tree/main/hub), in my repo) with a hosts file redirect to a `.localhost` domain.

**Reply by AlexErrant** [reply]
Ohh interesting, I just noticed that you specified `npx solid-start start`, not `npx solid-start dev`. Yeah I'm noticing this too with `start` on Node. Seems like a bug to me.
